client/vite-env.d.ts

```diff
@@ -13,6 +13,7 @@ interface ImportMetaEnv {
     readonly VITE_EQUIPMENT_IDS_WITHOUT_INPERSON: string;
     readonly VITE_MODULE_IDS_WITHOUT_INPERSON: string;
     readonly VITE_HELP_PAGE_URL: string;
+    readonly VITE_TRAINING_MAX_ATTEMPTS_PER_DAY_BEFORE_LOCK: string;
```

Comment:
I like the "you have 4 attempts left warning" but this env variable somewhat duplicated `TRAINING_MAX_ATTEMPTS_PER_DAY_BEFORE_LOCK` on the server. you have to add it here with the VITE prefix for the client to access it, but would you mind changing the uses of it on the backend to use the VITE prefix as well so we don't have to keep 2 different variables in sync

the client cant see server vars, but the server can see client vars when on production

Reply:
i just realized this is what stephen said and i agree with him more than myself so this env var shouldnt need to exist. 
I think you should stick to the backend as the source of truth and if you want to perserve the 'X tries', 'Y remaining' you should add those to the module query and have the backend fill them in bc the backennd already tracks that